ui/src/app/stores/workflows/workflows.effects.ts

```diff
@@ -117,4 +127,40 @@ export class WorkflowsEffects {
       }
     }),
   );
+
+  @Effect({ dispatch: true })
+  workflowDelete = this.actions.pipe(
+    ofType(WorkflowActions.DELETE_WORKFLOW),
+    switchMap((action: WorkflowActions.DeleteWorkflow) => {
+      return this.workflowService.deleteWorkflow(action.payload).pipe(
+        mergeMap((result: boolean) => {
```

Comment:
I think you can reduce one level of nesting here
```
     return this.workflowService.deleteWorkflow(action.payload)
},
mergeMap((result: boolean) => {

```

Reply:
I need that nesting because of action payload and I don't want to return tuple.

Reply:
Ah sorry didn't see that.